README.md

```diff
@@ -474,13 +482,26 @@ function getTrackerLoader(reactiveMapper) {
   };
 }
 
+function composeWithTracker(loadFunc) {
```

Comment:
I don't recommend using this.
Of course, you could use it.
But I don't really wanna put it on the README.